projects/1.18/assets/beyond-earth/beyond_earth/lang/zh_cn.json

```
@@ -0,0 +1,348 @@
+  "item.beyond_earth.engine_frame": "引擎框架",
+  "item.beyond_earth.ostrum_tank": "紫金燃油罐",
+  "item.beyond_earth.desh_engine": "戴斯制引擎",
+  "item.beyond_earth.hammer": "锤子",
+  "item.beyond_earth.ostrum_engine": "紫金制引擎",
+  "item.beyond_earth.alien_zombie_spawn_egg": "外星僵尸刷怪蛋",
+  "item.beyond_earth.ostrum_ingot": "紫金锭",
+  "item.beyond_earth.desh_tank": "戴斯燃油罐",
+  "item.beyond_earth.rocket_nose_cone": "火箭鼻锥",
+  "item.beyond_earth.compressed_steel": "致密钢板",
+  "item.beyond_earth.steel_ingot": "钢锭",
+  "item.beyond_earth.wheel": "轮子",
+  "item.beyond_earth.steel_tank": "钢罐",
+  "item.beyond_earth.pygro_spawn_egg": "派格洛刷怪蛋",
+  "item.beyond_earth.pygro_brute_spawn_egg": "残暴派格洛刷怪蛋",
+  "item.beyond_earth.iron_stick": "铁棍",
+  "item.beyond_earth.oxygen_gear": "氧气套装",
+  "item.beyond_earth.engine_fan": "引擎扇叶",
+  "item.beyond_earth.rocket_fin": "火箭尾翼",
+  "item.beyond_earth.rover": "漫游车",
+  "item.beyond_earth.alien_spawn_egg": "外星村民刷怪蛋",
+  "item.beyond_earth.cheese": "奶酪",
+  "item.beyond_earth.mogler_spawn_egg": "莫格洛刷怪蛋",
+  "item.beyond_earth.martian_raptor_spawn_egg": "火星凶兽刷怪蛋",
+  "item.beyond_earth.desh_ingot": "戴斯锭",
+  "item.beyond_earth.compressed_desh": "致密戴斯",
+  "item.beyond_earth.desh_nugget": "戴斯粒",
+  "item.beyond_earth.ostrum_nugget": "紫金粒",
+  "item.beyond_earth.steel_nugget": "钢粒",
+  "item.beyond_earth.water_pump": "水泵",
+  "item.beyond_earth.raw_desh": "粗戴斯",
+  "item.beyond_earth.raw_ostrum": "粗紫金",
+  "item.beyond_earth.raw_desh_block": "粗戴斯块",
+  "item.beyond_earth.raw_ostrum_block": "粗紫金块",
+  "item.beyond_earth.ice_shard": "冰刺",
+
+  "item.beyond_earth.rocket_t1": "一阶火箭",
+  "item.beyond_earth.rocket_t2": "二阶火箭",
+  "item.beyond_earth.rocket_t3": "三阶火箭",
+  "item.beyond_earth.rocket_t4": "四阶火箭",
+
+  "item.beyond_earth.oxygen_mask": "氧气面罩",
+  "item.beyond_earth.space_suit": "太空服",
+  "item.beyond_earth.space_pants": "太空裤",
+  "item.beyond_earth.space_boots": "太空靴",
+
+  "item.beyond_earth.netherite_oxygen_mask": "下界合金制氧气面罩",
+  "item.beyond_earth.netherite_space_suit": "下界合金制太空服",
+  "item.beyond_earth.netherite_space_pants": "下界合金制太空裤",
+  "item.beyond_earth.netherite_space_boots": "下界合金制太空靴",
+
+  "item.beyond_earth.jet_suit_oxygen_mask": "喷气套装氧气面罩",
+  "item.beyond_earth.jet_suit": "喷气背包",
+  "item.beyond_earth.jet_suit_pants": "喷气套装裤",
+  "item.beyond_earth.jet_suit_boots": "喷气套装靴",
+
+  "item.beyond_earth.calorite_ingot": "炽金锭",
+  "item.beyond_earth.compressed_calorite": "致密炽金",
+  "item.beyond_earth.calorite_nugget": "炽金粒",
+  "item.beyond_earth.raw_calorite": "粗炽金",
+  "item.beyond_earth.calorite_engine": "炽金制引擎",
+  "item.beyond_earth.calorite_tank": "炽金罐",
+
+
+  "itemGroup.tab_normal": "超越地平",
+  "itemGroup.tab_flags": "超越地平: 旗帜",
+  "itemGroup.tab_globes": "超越地平: 星球仪",
+  "itemGroup.tab_blocks": "超越地平: 方块",
+  "itemGroup.tab_basics": "超越地平: 基本",
+  "itemGroup.tab_machines": "超越地平: 机器",
+  "itemGroup.tab_spawn_eggs": "超越地平: 刷怪蛋",
+  "itemGroup.tab_materials": "超越地平: 材料",
```

模组是否有官方译名，这`超越地平`多少有点草率
顺带改点东西
```suggestion
  "itemGroup.tab_normal": "超越地平",
  "itemGroup.tab_flags": "超越地平丨旗帜",
  "itemGroup.tab_globes": "超越地平丨星球仪",
  "itemGroup.tab_blocks": "超越地平丨方块",
  "itemGroup.tab_basics": "超越地平丨基本",
  "itemGroup.tab_machines": "超越地平丨机器",
  "itemGroup.tab_spawn_eggs": "超越地平丨刷怪蛋",
  "itemGroup.tab_materials": "超越地平丨材料",
```

---

飞越寰宇丨地球之外